### ISSUE TITLE
dapp test results in InternalCompilerError when compiling contract that calls AAVE V2 contract functions

### ISSUE THREAD

**Issue by Ghorbanian** [dapphub/dapptools#625]
### Expected Behavior ###
---
Call to `dapp test`  builds contracts and tests and runs the tests, outputting the results.

### Current Behavior ###
---
Call to `dapp test`  results in the following error:
```
InternalCompilerError: 

hevm: (^?!): empty Fold
CallStack (from HasCallStack):
  error, called at src/Control/Lens/Fold.hs:1310:28 in lens-4.18.1-9qmhqm9T86YJHOPOEopx4d:Control.Lens.Fold
  ^?!, called at src/EVM/Solidity.hs:342:20 in hevm-0.44.1-LcTWU2RNn7iMuZpzDnXLB:EVM.Solidity
```

### Steps to Reproduce ###
---
Clone and run the minimal example at https://github.com/Ghorbanian/geb-saviour.

**Comment by Ghorbanian** [comment]
I should also include that the error comes up when attempting to call lendingpool.getReserveData on line 208 of /src/saviours/GeneralTokenReserveSafeSaviour.sol:
```
address aTokenAddress =
            lendingPool.getReserveData(ethReserveAddress).aTokenAddress;
```

Commenting out all of the code starting at that line to the end of the deposit function results in expected behavior for `dapp test`.

**Comment by MrChico** [comment]
as I mentioned in dapphub chat, this is an error in the Solidity compiler, not in dapptools. `InternalCompilerError` means that Solidity was unable to compile the contracts, https://github.com/ethereum/solidity/issues/9270. I believe this was fixed in solc 0.7.3
